src/Operation/ScanLeft.php

```diff
@@ -26,24 +26,26 @@ final class ScanLeft extends AbstractOperation
     /**
      * @pure
      *
-     * @return Closure(callable((T|null), T, TKey, Iterator<TKey, T>): (T|null)):Closure (T|null): Closure(Iterator<TKey, T>): Generator<int|TKey, T|null>
+     * @template V
+     *
+     * @return Closure(callable(V, T, TKey, Iterator<TKey, T>): V|T): Closure (V): Closure(Iterator<TKey, T>): Generator<TKey, V|T>
```

Comment:
this is out of sync with the interface. Here the callable returns `V|T`, but in the interface it's `V`

Reply:
Thanks! I updated it anyway, I guess it's better now.